docker.env

```diff
@@ -0,0 +1,12 @@
+NODE_ENV=development
```

Comment:
don't commit this file, use `RUN cp -p {desired_env}.env .env` in Dockerfile above